src/contracts/helpers/Sorter.sol

```diff
@@ -106,7 +106,7 @@ contract Sorter is AtlasConstants {
             return false;
         }
 
-        // solverOp.solver must not be the atlas or verification address
+        // Ensure the solver control address matches the configured dApp control address
         if (solverOp.solver == address(ATLAS) || solverOp.solver == address(VERIFICATION)) {
             return false;
         }
```

Comment:
The updated comment looks correct, but it should replace the one that mentions codehash, not this one which is correct in the context of the code below it. Will fix quickly